cmake/modules/FindLibR.cmake

```diff
@@ -107,20 +107,31 @@ else()
       if(NOT LIBR_HOME)
 
          # read home from the registry
-         get_filename_component(LIBR_HOME
-            "[HKEY_LOCAL_MACHINE\\SOFTWARE\\R-core\\R;InstallPath]"
-            ABSOLUTE CACHE)
+         get_filename_component(LIBR_HOME "[HKEY_LOCAL_MACHINE\\SOFTWARE\\R-core\\R;InstallPath]" ABSOLUTE)
```

Comment:
We shouldn't cache the variable until we're certain we found the "right" one, so I removed the cache step and pushed it down later.